osf/models/registrations.py

```diff
@@ -1261,25 +1261,32 @@ def create_from_node(cls, user, schema, node=None, data=None, provider=None):
         else:
             provider.validate_schema(schema)
 
-        if not node:
-            # If no node provided, a DraftNode is created for you
-            node = DraftNode.objects.create(creator=user, title='Untitled')
+        excluded_attributes = []
+        if node:
+            branched_from = node
+        else:
+            branched_from = DraftNode.objects.create(creator=user, title='Untitled')
+            excluded_attributes = ['affiliated_institutions']
 
-        if not (isinstance(node, Node) or isinstance(node, DraftNode)):
+        if not isinstance(branched_from, (Node, DraftNode)):
             raise DraftRegistrationStateError()
 
         draft = cls(
             initiator=user,
-            branched_from=node,
+            branched_from=branched_from,
             registration_schema=schema,
             registration_metadata=data or {},
             provider=provider,
         )
         draft.save()
-        draft.copy_editable_fields(node, Auth(user), save=True)
+        draft.copy_editable_fields(
+            branched_from,
+            excluded_attributes=excluded_attributes
```

Comment:
do we need this? I would assume that a DraftNode just has zero Affiliations and therefore wouldn't copy anything?

Reply:
You're right we don't need it here.